.github/workflows/ci.yml

```
@@ -14,8 +14,16 @@ jobs:
     runs-on: ${{ matrix.platform }}
     timeout-minutes: 60
     steps:
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 17: could not find expected ':'

(syntax-check)